loader/src/sources/cdc/api.js

```diff
@@ -462,7 +474,7 @@ async function checkAvailability(handler, options) {
 
     const stores = Object.values(entriesByStoreId);
     const formatted = stores
-      .map(formatStore)
+      .map((store) => formatStore(store, checkedAt))
       .filter(Boolean)
       .map(markUnexpectedCvsIds);
 
```

Comment:
A couple of loader unit tests with the new store number formats would probably be a good idea, but obviously up to you as to how needed/not needed that is